patches/chrome-browser-resources-bookmarks-toolbar.html.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/resources/bookmarks/toolbar.html b/chrome/browser/resources/bookmarks/toolbar.html
+index df0329980236ff0b88b70a8763c6373ec61caddc..01104801a796e82d4d4588c309ca0deadbd4c116 100644
+--- a/chrome/browser/resources/bookmarks/toolbar.html
++++ b/chrome/browser/resources/bookmarks/toolbar.html
+@@ -1,6 +1,6 @@
+ <link rel="import" href="chrome://resources/html/polymer.html">
+ 
+-<link rel="import" href="chrome://resources/cr_elements/cr_toolbar/cr_toolbar.html">
```

Comment:
can we just patch cr_toolbar.html to import br_toolbar.html and add html comments around the original source? Then we're only patching in one place and it will automatically apply to any files that use cr_toolbar